stores/bookshelf.ts

```diff
@@ -9,12 +9,15 @@ export const useBookshelfStore = defineStore('bookshelf', () => {
   const hasFetched = ref(false)
   const nextKey = ref<number | undefined>(undefined)
 
-  const items = computed(() => nftClassIds.value.map(nftClassId => ({ id: nftClassId, nftClassId })))
-
   const getNFTsByNFTClassId = computed(() => (nftClassId: string) => {
     return Object.values(nftByNFTClassIds.value[nftClassId] || {})
   })
 
+  const items = computed(() => nftClassIds.value.map(nftClassId => ({
```

Comment:
I think more robust to directly use `nftByNFTClassIds` to compute `items` and `nftClassIds`